app/controllers/unattended_controller.rb

```diff
@@ -64,9 +70,8 @@ def render_template(type)
       logger.debug "rendering DB template #{config.name} - #{type}"
       safe_render config
     else
-      msg = "unable to find #{type} template for [#{@host.name}] running [#{@host.operatingsystem}]"
-      logger.error msg
-      render :text => msg, :status => :not_found
+      error_message = N_("unable to find %{type} template for %{host} running %{os}") % {:type => type, :host => @host.name, :os => @host.operatingsystem}
```

Comment:
Fixed, on second thought, unattended controller have i18n with defaults I guess (no session), but this will work with preview I guess.